Modules/_randommodule.c

```diff
@@ -259,7 +263,7 @@ random_seed_time_pid(RandomObject *self)
     key[0] = (uint32_t)(now & 0xffffffffU);
     key[1] = (uint32_t)(now >> 32);
 
-#ifdef MS_WINDOWS_NON_DESKTOP
+#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_DESKTOP)
```

Comment:
The CRT defines `_CRT_USE_WINAPI_FAMILY_DESKTOP_APP` for the `DESKTOP` and `SYSTEM` API partitions in "corecrt.h". This macro determines whether or not `_getpid()` is defined.

```suggestion
#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_DESKTOP) && !defined(MS_WINDOWS_SYSTEM)
```